README.md

```diff
@@ -1,56 +1,55 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Catalog of my things
 
-> Description the project.
+> App developed in Ruby with UI to manage Books, Music and Games.
+![Screen Shot 2022-11-17 at 12 04 51](https://user-images.githubusercontent.com/99816838/202528337-5dc461ac-fb4e-480b-a098-f22085db67b4.png)
+![Screen Shot 2022-11-17 at 12 07 02](https://user-images.githubusercontent.com/99816838/202528352-e512ff4d-990c-42d5-88c7-99896eda4317.png)
+![Screen Shot 2022-11-17 at 12 08 13](https://user-images.githubusercontent.com/99816838/202528366-c6c15fe5-11db-4070-8e37-bc7e0341180e.png)
+![Screen Shot 2022-11-17 at 12 10 10](https://user-images.githubusercontent.com/99816838/202528376-39622165-8693-40d6-afeb-7ed7f84bd170.png)
 
 
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
 
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+## Built With
 
+- Ruby
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+Verify you have Ruby installed on your system
 
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
+### Clone the repository wit this link https://github.com/Flunavgt/catalog_of_my_things.git
 
-### Setup
+### Change to repo directory
 
-### Install
+### run App.rb
 
-### Usage
+## Video description [Here](https://drive.google.com/file/d/12nhbyvVRRGvZfIU4RXaDwh00u4ALkPo7/view?usp=sharing)
```

Comment:
- [ ] Nice job adding video link to the ReadMe, however, as informed in the previous reviews your video should not have length more than **5 minutes**, which your video is currently 7+ minutes long. I'll suggest you edit your video to leaving the most important parts so that you can achieve the said length, in case the team cannot remake a new shorter video soon.